lib/rules/no-redundant-landmark-role.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const Rule = require('./base');
+const AstNodeInfo = require('../helpers/ast-node-info');
+
+function createErrorMessage(element) {
+  return `Use of redundant or invalid role on <${element}> detected. If a landmark element is used, any role provided will either be redundant or incorrect.`;
+}
+
+const LANDMARK_ROLES = [
```

Comment:
Can you add a link to https://www.w3.org/TR/html-aria/#docconformance just above here?